### PR TITLE
Version next package information

### PR DESCRIPTION
**Pull request by Wagnerp**

### CODE REVIEW

File: Source/Krypton Toolkit/Examples/ComboBoxExtApp/ComboBoxExtApp.csproj [hunk 1]
```diff
@@ -8,7 +8,7 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <ProjectReference Include="..\..\Main\Krypton.Toolkit.Suite.Extended.ComboBoxExt\Krypton.Toolkit.Suite.Extended.ComboBoxExt.csproj" />
+        <ProjectReference Include="..\..\Main\Krypton.Toolkit.Suite.Extended.ComboBoxExt\Krypton.Toolkit.Suite.Extended.Combo.Box.csproj" />
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Examples/ComboBoxExtApp/ComboBoxExtApp.csproj:11]: I do not like these changes>
DataGridView is a single named entity, and the namespace should reflect that.

File: Source/Krypton Toolkit/Examples/DataGridView/DataGridViewExt.csproj [hunk 2]
```diff
@@ -17,7 +17,7 @@
     </ItemGroup>
 
     <ItemGroup>
-        <ProjectReference Include="..\..\Main\Krypton.Toolkit.Suite.Extended.DataGridViewExt\Krypton.Toolkit.Suite.Extended.DataGridViewExt.csproj" />
+        <ProjectReference Include="..\..\Main\Krypton.Toolkit.Suite.Extended.DataGridViewExt\Krypton.Toolkit.Suite.Extended.Data.Grid.View.csproj" />
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Examples/DataGridView/DataGridViewExt.csproj:20]: Name change > see previous comment

File: Source/Krypton Toolkit/Examples/ComboBoxExtApp/ComboBoxExtApp.csproj (hunk 1 above)
Comment (Wagnerp) [reply Source/Krypton Toolkit/Examples/ComboBoxExtApp/ComboBoxExtApp.csproj:11]: Ok, will change to all one word. I don't think we need the `Ext`, as the extensions to them are already implied in the namespace.

File: Source/Krypton Toolkit/Examples/DataGridView/DataGridViewExt.csproj (hunk 2 above)
Comment (Wagnerp) [reply Source/Krypton Toolkit/Examples/DataGridView/DataGridViewExt.csproj:20]: See previous comment

**Pull request merged**